CONTRIBUTING.md

```diff
@@ -17,11 +17,45 @@ git push -f
 ```
 
 It's okay to force push over your own PR branch. In fact, that's what we want so that the commits are clean.
-
+This also means you shouldn't use `git pull` and `git merge` when contributing.
 
 ## How to squash a PR
-See [this how to](https://twitter.com/stephenfluin/status/1009904095073718275).
```

Comment:
We should probably start with the recommendation to use `git commit --amend` while working on your PR first. Avoiding multiple commits is much cleaner and easier than fixing them.